docker-compose.yml

```diff
@@ -1,306 +1,441 @@
-            - ./config/express-common.env
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    photo-audio-haptics-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-photo-audio-haptics:unstable
-        env_file:
-            - ./config/express-common.env
-        depends_on:
-            - supercollider
-            - espnet-tts
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-        volumes:
-            - sc-store:/tmp/sc-store
-    ocr-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-ocr:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    motd-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-motd:unstable
-        environment:
-            - MOTD=Hello, world!
-        env_file:
-            - ./config/express-common.env
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    high-charts-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-high-charts:unstable
-        env_file:
-            - ./config/express-common.env
-        depends_on:
-            - supercollider
-            - espnet-tts
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-        volumes:
-            - sc-store:/tmp/sc-store
-
-    svg-od-handler:
-         image: ghcr.io/shared-reality-lab/image-handler-svg-od:unstable
-         labels:
-             ca.mcgill.a11y.image.handler: enable
-
-
-    osm-streets-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-osm-streets:unstable
-        env_file:
-            - ./config/express-common.env
-        depends_on:
-            - supercollider
-            - espnet-tts
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-        volumes:
-            - sc-store:/tmp/sc-store
-    svg-semantic-seg-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-svg-semantic-seg:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    svg-open-street-map-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-svg-open-street-map:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-            ca.mcgill.a11y.image.port: 5000
-    svg-depth-map:
-        image: ghcr.io/shared-reality-lab/image-handler-svg-depth-map:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    photo-tactile-svg-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-photo-tactile-svg:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    map-tactile-svg-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-map-tactile-svg:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
-    svg-action-recognition-handler:
-        image: ghcr.io/shared-reality-lab/image-handler-svg-action-recognition:unstable
-        labels:
-            ca.mcgill.a11y.image.handler: enable
+  image-unicorn-cim-mcgill-ca:
```

Comment:
Since this and the pegasus one (and goaccess) are only for running the website, and have nothing to do with running a working IMAGE server, would it be cleaner to remove these from the core docker-compose.yml? This is likely to only cause confusion with external people trying to understand how to run a server. Can be broken out into a separate issue if you want to deal with it separately.

Reply:
Looking again now, I am wrong on this, since these traefik sections also do the routing of all of the preprocessor requests, etc, correct? So it might still make sense to break these out, but they do rely on orchestrator, etc. so they are intertwined. I'm not sure which way to go on this. Is it worth separating them out as separate docker-compose files, or is this more trouble than it is worth since they are in fact connected? How would we want this to be segmented if we were an external team bringing up our own server, for which we might not want a website at all? Would it be clear what we should do, or is there in fact a better way to organize this?